CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ##### Enhancements
 
+* None
+
+##### Bug Fixes
+
+* Addressed issue where scrolling to the focused text field would not work properly.
+[Daniel Larsen](https://github.com/grandlarseny)
+[#48](https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/48)
+
+## 2.1.1 (2020-01-16)
```

Comment:
Version `2.1.1` hasn't been released yet, and when it is released, it will include your bug fix. Could you please restructure this `CHANGELOG` entry so that it reads like this?

```
## Master

##### Enhancements

* Added Carthage support.
[Ryan Gant](https://github.com/ganttastic)
[#46](https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/46)

##### Bug Fixes

* Addressed issue where scrolling to the focused text field would not work properly.
[Daniel Larsen](https://github.com/grandlarseny)
[#48](https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/48)

## 2.1.0 (2019-04-30)

##### Enhancements

// ...
```

Reply:
Sure thing